src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
+        public const int CbAlign = 16;
```

Comment:
This shouldn’t be here.  This is Sse specific, but CpuMathUtils should be agnostic of the ISA.

Reply:
Maybe if you name it ‘SseCbAlign’, that would work.

Reply:
Agree and thanks!  The only current call site of `CbAlign` is in this file:
https://github.com/dotnet/machinelearning/blob/b51d9f9060acfcbe84405ce37f9c045654573ee0/src/Microsoft.ML.Transforms/RffTransform.cs#L24-L29
where Line 26 should be changed to `using CpuMathUtils;`, and the bunch of `CpuUtils.CbAlign` in subsequent lines should be changed to `CpuMathUtils.SseCbAlign` for the moment.  While this change may expose the ISA-specific element (the `Sse` prefix of `SseCbAlign`) out of `CpuMathUtils`, I think it's good for now.

In the future when we also support AVX, should I change `public const int CbAlign = 16;` to something like the code below?
```
public const int CbAlign = Avx.IsSupported ? 32 : 16;
**OR**
public int GetCbAlign() => Avx.IsSupported ? 32 : 16;
```